bin/GameIndex.dbf

```diff
@@ -483,8 +483,8 @@ Name   = Gran Turismo 4 - Prologue
 Region = NTSC-Unk
 Compat = 5
 //eeClampMode = 3 // Text in races works
-vuClampMode = 2 // Text in GT mode works
-MemCardFilter = SCAJ-20066/SCAJ-30006/SCAJ-30007/SCAJ-30008/SCPS-15055/SCPS-17001/SCPS-19252/SCPS-19304/SCPS-15009/SCPS-55007
+//vuClampMode = 2 // Text in GT mode works
+//MemCardFilter = SCAJ-20066/SCAJ-30006/SCAJ-30007/SCAJ-30008/SCPS-15055/SCPS-17001/SCPS-19252/SCPS-19304/SCPS-15009/SCPS-55007
```

Comment:
Why did you comment out memcard filters?

Reply:
GT tested with a memcardfiltered game and he can't load any saves. While a friend of him has the Korean version which doesnt have that line and can load saves as normal.

Reply:
I re-enabled the memcard filters again to be checked again later in the future.